grails-app/domain/org/pih/warehouse/core/Location.groovy

```diff
@@ -161,13 +160,17 @@ class Location implements Comparable<Location>, java.io.Serializable {
 
     }
 
+    Boolean isManagedLocally() {
```

Comment:
This is a great idea. You could have just used isLocal() or getLocal() so you didn't have to change all "local" references to "managedLocally". However, if you made the change to be more readable (since the meaning of "local" is ambiguous) then I support that decision.

Reply:
yes, I changed it to be more clear.